frontend/model/contracts/chatroom.js

```diff
@@ -114,6 +114,18 @@ function messageReceivePostEffect ({
   shouldSoundMessage && sbp('okTurtles.events/emit', MESSAGE_RECEIVE)
 }
 
+function deleteEncryptedFilesByToken (manifestCids: string | string[], innerSigningContractID: string) {
+  if (typeof manifestCids === 'string') {
+    manifestCids = [manifestCids]
+  }
+
+  if (Array.isArray(manifestCids) && manifestCids.length) {
+    sbp('gi.actions/identity/removeFiles', {
```

Comment:
This could reject, and that rejection isn't handled.

Reply:
Rejection can't be handled here. Already handled the errors in `gi.actions/identity/removeFiles` function.

Reply:
> Already handled the errors in gi.actions/identity/removeFiles function.

The exception does need to be handled here too. Please see [my comment here about that.](https://github.com/okTurtles/group-income/pull/1892#discussion_r1557878667)